contracts/cw20-streams/src/error.rs

```diff
@@ -28,7 +28,7 @@ pub enum ContractError {
 
     // TODO: Implement refund
```

Comment:
Nice name change.
BTW, if you return an error, you abort the transaction and all state changes (including sending the tokens to this contract) get reverted. So if it is a same-tx refund, no need to do anything besides error (which is done already)